day02/ex10/ex10.md

```diff
@@ -0,0 +1,108 @@
+ ### Exercise 10 - Confusion Matrix
+
+|                         |                         |
+| -----------------------:| ----------------------- |
+|   Turning directory :   |  ex10                   |
+|   Files to turn in :    |  confusion_matrix.py    |
+|   Forbidden function :  |  None                   |
+|   Remarks :             |  n/a                    |
+
+**Objectives:**
+
+The goal of this exercise is to recreate the function confusion_matrix of sklearn.metrics and to learn what represents
+the confusion matrix.
+
+**Instructions:**
+
+For the sake of simplicity, we will only ask you to have three parameters.
+Be careful to respect the order, true labels are rows and predicted labels are columns !
```

Comment:
What do you mean more precisely ? Are you saying that Y_pred is a vector (Nx1) and  Y_true a  matrix of 1xN ?

Reply:
What I mean is this, I will add this code sniplet :
```
#                      [predicted labels]    
#                       label_1  label_2
#   [ true ]  label_1         .        .
#   [labels]  label_2         .        .
```
Check [this link.](https://miro.medium.com/max/1194/0*wKaznIJzZF54b87B.jpg)